StudyHub/Entities/OpenEndedOption.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace StudyHub.Entities;
+
+public class OpenEndedOption : AssignmentTaskOptionBase
+{
```

Comment:
Use common code style for all classes to keep code style consistent
_Use style like Admin class uses_